src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
```

Comment:
In general, I use .d.ts files for custom [Utility Types](https://www.typescriptlang.org/docs/handbook/utility-types.html), and create specific types such as entities(User, Post, Comment)... in simple .ts files from which I import them. So, I recommend you to do the same